lib/chat/types.js

```diff
@@ -161,11 +161,11 @@ export type ChatOptionsSearchRegexp = ChatOptionsSearchInbox & {|
 /**
  */
 export type AnyChatOption =
```

Comment:
Not sure we need `AnyChatOption` or the Chat API Function Signatures anymore. I can delete but lemme know

Reply:
Destroy at will